src/tests/invalid_config/bad_git_url.toml

```
@@ -0,0 +1,23 @@
+[project]
```

Running this example, I think? I'd expect a better error message than:
```
Failed to load config at `.`

Caused by:
    TOML parse error at line 19, column 16
       |
    19 | dependencies = [
       |                ^
    data did not match any variant of untagged enum ConfigDependency
```

---

yeah... sadly serde isn't great for that. I need to see if i can propagate the error better

---

so the `untagged` just swallows all inner errors :/ We might write a custom visitor that shows it but let's keep that for later